Add support for Initialisation Scripts with rcS

mlsmaycon opened the pull request:
This PR aims to support OS images that doesn't have full support for sysV and relies on rcS to start their services
* It checks for rcS in inittab
* uses the installed script for service related commands

As I had problem, I added one small change to the systemd check:
* check if systemctl command is available (harmless but helpful for chroot with bind /proc)

kardianos commented:
I merged a more targeted change to fix windows.  Can look at other changes in isolation.

mlsmaycon commented:
> I merged a more targeted change to fix windows. Can look at other changes in isolation.

Alright @kardianos, I've updated my fork and removed the windows fix.
